scenario/prechecks.yaml

```diff
@@ -28,3 +28,4 @@ preparation:
           - check_dst_sql: True
           - check_dst_rabbit: True
       - check_networks: True
+  - check_users_availability: True
```

Comment:
Is it possible to keep them ? As one used for stage resource migration and another just can be used for testing CF. It is easier from document prospective.